pkg/types/040/types.go

```diff
@@ -304,3 +310,49 @@ func (c *IPConfig) UnmarshalJSON(data []byte) error {
 	c.Gateway = ipc.Gateway
 	return nil
 }
+
+type Route struct {
```

Comment:
@muff1nman Let's acutally just use the 020 implementation since they are the same. 040 types import the 020 so it's fine to do that.

Reply:
Nevermind... do what Casey suggests with the `MTU *uint`